docs/en/observability/tail-logs.asciidoc

```diff
@@ -14,6 +16,19 @@ along with the power of search.
 Click *Stream Live* to view a continuous flow of log messages in real time, or
 click *Stop streaming* to view historical logs from a specified time range.
 
+[discrete]
+[[activate-logs-stream]]
+== Activate Logs Stream
+
+Because <<explore-logs, Logs Explorer>> is replacing Logs Stream, Logs Stream and the Logs Stream dashboard panel are disabled by default.
+To activate Logs Stream and the Logs Stream dashboard panel complete the following steps:
+
+. Go to **Management** → **Advanced Settings**
```

Comment:
```suggestion
. Go to **Management** → **Advanced Settings**.
```

Reply:
Added this while on a call with Edu to show him how suggestions work. We can ignore.